helpers/deploymentConfig.ts

```diff
@@ -2303,6 +2303,13 @@ export const globalConfig: NetworkConfig = {
         decimals: 18,
         tokenAddress: ethers.constants.AddressZero,
       },
+      {
+        isMock: true,
```

Comment:
Is it a mock token?

Reply:
No It is not a mock token, Anyway I removed it because the XVS token configuration already exists there. [f77f40a](https://github.com/VenusProtocol/isolated-pools/pull/331/commits/f77f40a81d4a1509a338e4ac624cf0fced787100)